Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:22
+
+RUN npm i -g pnpm
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
should the command be `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`

https://www.rust-lang.org/tools/install

Reply:
That command initiated a cli based interactive installer. I tried a bunch of flags and ended up with this as the simplest method.